### PR TITLE
chore: migrate from btc-pk to eots-pk in fpd related commands and docs

### PR DESCRIPTION
**Pull request by RafilxTenfen**
Only modifies naming of flags, no logic is modified

### CODE REVIEW

File: finality-provider/cmd/fpd/daemon/daemon_commands.go [hunk 1]
```diff
@@ -217,7 +217,7 @@ func runCommandLsFP(cmd *cobra.Command, args []string) error {
 // CommandInfoFP returns the finality-provider-info command by connecting to the fpd daemon.
 func CommandInfoFP() *cobra.Command {
 	var cmd = &cobra.Command{
-		Use:     "finality-provider-info [fp-pk-btc-hex]",
+		Use:     "finality-provider-info [fp-pk-eots-hex]",
```
Comment (gitferry) [inline finality-provider/cmd/fpd/daemon/daemon_commands.go:220]:
```suggestion
		Use:     "finality-provider-info [fp-eots-pk-hex]",
```

File: finality-provider/cmd/fpd/daemon/daemon_commands.go [hunk 2]
```diff
@@ -257,7 +257,7 @@ func runCommandInfoFP(cmd *cobra.Command, args []string) error {
 // CommandRegisterFP returns the register-finality-provider command by connecting to the fpd daemon.
 func CommandRegisterFP() *cobra.Command {
 	var cmd = &cobra.Command{
-		Use:     "register-finality-provider [fp-pk-btc-hex]",
+		Use:     "register-finality-provider [fp-pk-eots-hex]",
```
Comment (gitferry) [inline finality-provider/cmd/fpd/daemon/daemon_commands.go:260]: ditto

File: finality-provider/cmd/fpd/daemon/daemon_commands.go [hunk 3]
```diff
@@ -305,7 +305,7 @@ func runCommandRegisterFP(cmd *cobra.Command, args []string) error {
 // CommandAddFinalitySig returns the add-finality-sig command by connecting to the fpd daemon.
 func CommandAddFinalitySig() *cobra.Command {
 	var cmd = &cobra.Command{
-		Use:     "add-finality-sig [fp-pk-btc-hex] [block-height]",
+		Use:     "add-finality-sig [fp-pk-eots-hex] [block-height]",
```
Comment (gitferry) [inline finality-provider/cmd/fpd/daemon/daemon_commands.go:308]: ditto

**Review by gitferry — APPROVED** [review]

File: finality-provider/cmd/fpd/daemon/daemon_commands.go (hunk 3 above)
Comment (RafilxTenfen) [reply finality-provider/cmd/fpd/daemon/daemon_commands.go]: https://github.com/babylonchain/finality-provider/pull/524/commits/63009efadf97f7b092d7f712fc1f90aabc6f6633

File: finality-provider/cmd/fpd/daemon/daemon_commands.go (hunk 2 above)
Comment (RafilxTenfen) [reply finality-provider/cmd/fpd/daemon/daemon_commands.go]: https://github.com/babylonchain/finality-provider/pull/524/commits/63009efadf97f7b092d7f712fc1f90aabc6f6633

File: finality-provider/cmd/fpd/daemon/daemon_commands.go (hunk 1 above)
Comment (RafilxTenfen) [reply finality-provider/cmd/fpd/daemon/daemon_commands.go]: https://github.com/babylonchain/finality-provider/pull/524/commits/63009efadf97f7b092d7f712fc1f90aabc6f6633

**Pull request merged**